frontend/client/enums.ts

```diff
@@ -30,6 +30,24 @@ export enum MiddlewareDeploymentStatus {
   DELETED = 6,
 }
 
+/** @note statuses where middleware deployment is moving from stopped to deployed, or vice versa, used for loading fallbacks */
+export const MiddlewareTransitioningStatuses = [
+  MiddlewareDeploymentStatus.DEPLOYING,
+  MiddlewareDeploymentStatus.STOPPING,
+];
+
+/** @node statuses where middleware deployment is running */
+export const MiddlewareRunningStatuses = [
+  MiddlewareDeploymentStatus.DEPLOYED,
+  ...MiddlewareTransitioningStatuses,
+];
+
+/** @note statuses where middleware is in the process of building/creating a new deployment */
```

Comment:
can ignore, but the comment is a bit confusing. E.g. `MiddlewareDeploymentStatus.DEPLOYING` sounds more like a status representing `in the process of building/creating`

Reply:
agreed, however it represents "starting" from both built and stopped